### ISSUE TITLE
Maximized windows prevent focus other windows

### ISSUE THREAD

**Issue by davidpa9708** [esjeon/krohnkite#138]
**Symptom**
I'm not really sure if this is a feature or a bug, but I noticed that when using Krohnkite, maximized applications also are kept above others. I expect them not to do that by default, or if that is the desired behavior at least have some setting to change it. 

**How to Reproduce**
1. Set `Tile` layout
2. Open 2 applications.
3. Maximize application 1
4. Change focus to application 2
4. Don't see application 2 onscreen since application 1 kept above other

**Expected behavior**
windows to not be kept above others while maximized

**Environment**
 - Distro: Kubuntu 20.10
 - KWin version: 5.19.5
 - Krohnkite version: 0.8, b9d3b2f
 - List of KWin scripts in use: Krohnkite

**Notes**
Here's a [video](https://user-images.githubusercontent.com/24394426/104132035-bedf7b80-5348-11eb-917f-d9a3b6cbd846.mp4) of it.
And another [one](https://user-images.githubusercontent.com/24394426/104132129-62c92700-5349-11eb-9269-90088a2868c7.mp4) showing how its `kept above others` when maximized
Good work with Krohnkite btw :+1:

**Comment by ItsAtNight** [comment]
**I can add to that:**
1. Set any layout (except Floating)
2. Switch to _Floating Layout_
3. Maximize a window

**Behavior**
Every other window (already open AND opened afterwards) is now behind the previously maximized window, even if it is made smaller again. The issue disappears when a window is maximzed once (only for that specific window) and reappears when switching to another layout and back.

**Environment**
- Distro: Debian 11 Bullseye (testing)
- KWin version: 5.20.5
- Krohnkite version: 0.8, b9d3b2f
- The only script is Kröhnkite

**Comment by ben2talk** [comment]
Yes, this is an issue with me using GIMP (after maximising, preferences opens behind the main window and I can't pull a dolphin window to the front to drag/drop) also with Firefox which I run with no titlebar - so it's not easy to unset 'always on top'.

Why do windows get set 'always on top'? I can't find anything in the settings dialogue...

**Comment by dbryce712** [comment]
This got fixed with the latest commit 2a47753fa2a37a9035116df4a7c2d73044373d82
